vars/neoDeploy.groovy

```diff
@@ -148,34 +146,30 @@ def call(parameters = [:]) {
             deployAccount = utils.getMandatoryParameter(configuration, 'account')
         }
 
-        def neo = new ToolDescriptor('SAP Cloud Platform Console Client', 'NEO_HOME', 'neoHome', '/tools/', 'neo.sh', null, 'version')
-        def neoExecutable = neo.getToolExecutable(this, configuration)
-        def neoDeployScript = """#!/bin/bash
-                                 "${neoExecutable}" ${warAction} \
+        def shCmdOptions = """${warAction} \
```

Comment:
[naming: shCmdOptions] 
* maybe we should keep 'neo' in the name, since it is about a neo command.
* It is not only about options. 'argument' would be a more generic term.

What about neoCmdArgs, just as a proposal?